account_cash_basis_group_base_line/README.md

```diff
@@ -0,0 +1,5 @@
+account_cash_basis_group_base_line
```

Comment:
@GavyMG 
Could you change the readme to new fashion using a reference to a folder:
 - https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

Reply:
Sorry @GavyMG
I wasn't clear here

New standard of OCA is auto-filling the `module/README.rst`
We just need to create the `module/readme/` folder with the following files `*.rst`:
 - https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

Please